packages/format-message-estree-util/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-var MODULE_NAME_PATTERN = /(^|\/)format-message$/
+var MODULE_NAME_PATTERN = /(^|\/|\.|-)format-message$/
```

Comment:
At this point, seems like we could simplify by just doing `/\bformat-message$/`

Reply:
Good catch!